guides/v2.1/cloud/project/new-relic.md

```diff
@@ -17,13 +17,47 @@ New Relic APM provides the following features to Magento:
 -  **App Map**—View all application dependencies within your Magento site, extensions, and external services.
 -  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event.
 
-Your {{site.data.var.ece}} account includes the agent software for New Relic; you do not need to purchase or install the New Relic extension (different than the APM service). You receive your credentials and license information from Magento during launch and can access your license key in the _Project Web Interface_ by clicking **View Details** for your project. The **NewRelic Service** section includes your Account Number, License Key, and other access keys.
+By default, the {site.data.var.ece}} Pro plan includes New Relic on the Staging and Production environments. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation. For {{site.data.var.ece}} Pro plan customers also need to submit a support ticket to use New Relic on an Integration branch. 
```

Comment:
"Customers with {{site.data.var.ece}} Pro plan will also need to..."

Not sure if it should be "customers with..." or "{{site.data.var.ece}} Pro plan customers..." but the original sentence definitely has an error